subsys/net/lib/http/http_server_http1.c

```diff
@@ -254,8 +269,12 @@ static int http1_send_headers(struct http_client_ctx *client, enum http_status s
 		return -EINVAL;
 	}
 
+	const char *header_template = is_client_http10(client)
```

Comment:
Please introduce variables at the beginning of the block (so at the start of the function in this case)